pkg/skuba/actions/cluster/init/manifests.go

```diff
@@ -874,6 +874,22 @@ spec:
                   fieldPath: spec.nodeName
           command:
             - /usr/bin/kured
+          livenessProbe:
+            httpGet:
+              path: /metrics
+              port: 8080
+            # The initial delay for the liveness probe is intentionally large to
+            # avoid an endless kill & restart cycle if in the event that the initial
+            # bootstrapping takes longer than expected.
+            initialDelaySeconds: 120
+            failureThreshold: 10
+            periodSeconds: 60
+          readinessProbe:
```

Comment:
readinessProbe a MUST?

Reply:
not strictly i'd say, as kured doesn't have any dependent services. but if we ever do RollingUpdates it might make sense for kubernetes to know when to shut down the old pod (when the /metrics endpoint of the new pod is ready)